app/models/story.rb

```diff
@@ -212,6 +214,19 @@ def v4?
     app_version == PRX::APP_VERSION
   end
 
+  # for each distribution, create a story distribution
+  def create_story_distributions
+    return unless series
+    series.distributions.each do |distro|
+      begin
+        distro.create_story_distribution(self)
+      rescue StandardError => err
+        logger.error(err)
+        NewRelic::Agent.notice_error(err)
```

Comment:
I like this.  Any case for also adding this to the series, when it's creating the feeder Podcast?